.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,77 @@
+# Contributing to _WPStarter_
+
+**First**, let us say: Thank you for contributing! :+1:
+
+The outlined steps in this document are just guidelines, not rules, 
+use your best judgment and feel free to propose changes to this document 
+in [a new pull request](https://github.com/wecodemore/wpstarter/compare).
+
+---
+
+## What should I know before getting started?
+
+In general, it's the following rule:
+
+> Try to play nice with later developers or people using this package.
+
+### Submitting changes
+
+1. Please open [an issue](https://github.com/wecodemore/wpstarter/issues/new) 
+   to discuss your changes before filing a pull request.
+2. When you send a pull request, we will love you forever.
+3. Always add changes to **a new branch** named 
+
+        git checkout -b issue-<issue number>-<topic>
+        # Example
+        # issue-54-feature-title
+
+    An exception are extremely minor changes. Those can go directly to `dev`  
+
+4. Always **file PRs against the `dev` branch**. Nothing goes to `master` without 
+   going to `dev` first. In case you did not do that, please just update/rebase 
+   your pull request.
+5. If you include tests, we will love you in all eternity. 
+   **Hint:** We can always use more test coverage.
+6. When you send a pull request, please try to explain your changes. A list of 
+   keywords is enough.
+
+We will label your issue or PR accordingly so that you can filter the list of 
+issues with the least effort possible. We also assign issues to milestones and 
+release minor or major versions when the list of issues exceeds a certain 
+threshold.
+
+### Commit changes
```

Comment:
I'm wondering if Git supports a `.commitmessage` or `.committemplate` template *per repo*, instead of just at the global level. If so, that might be a better way. Here's mine:

```
Component: Brief summary, less than 50 chars, no full stop

Longer description with more details, such as a `new_hook` being introduced with the context of a `$post` and a `$screen`. Use bullet points if necessary.

More paragraphs can be added as needed.

Fixes #9999. See #9999.
```